src/index.js

```diff
@@ -0,0 +1,81 @@
+/* eslint-disable no-restricted-globals */
+/* eslint-disable import/no-cycle */
+/* eslint-disable import/prefer-default-export */
+import './style.css';
+import { getTask } from './data';
+import { updateTask, deleteTask, deleteOne } from './controllers';
+import { dragDrop } from './dragDrop';
+
+export const taskList = document.querySelector('.task-list-container');
+
+function getInputValue(task) {
+  return task.description;
+}
+
+getTask().forEach((task) => {
+  taskList.innerHTML += `<li class="container task flex-center" draggable="true">
+  <span class="left flex-center">
+    <input id=${task.index} type="checkbox" ${
+  task.completed ? 'checked' : ''
+}  class="checkbox" />
+    <form class="edit-form" action="/">
+    <input data-index-number=${task.index} value='${getInputValue(
+  task,
+)}' class="${task.completed ? 'edit-task disabled' : 'edit-task'}" ${
+  task.completed ? 'disabled' : ''
+} ></form>
+  </span>
+  <span class="right">
+    <i class="fas fa-ellipsis-v"></i>
+    <i class="far fa-trash-alt"></i>
+  </span>
+</li>`;
+});
+
+export const task = document.querySelectorAll('.task');
+export const editTask = document.querySelectorAll('.edit-task');
+export const checkbox = document.querySelectorAll('.checkbox');
+const editForm = document.querySelectorAll('.edit-form');
+const reload = document.querySelector('.reload');
+
+editForm.forEach((form) => {
+  form.addEventListener('submit', (e) => {
+    e.preventDefault();
+    editTask.forEach((taskList) => {
+      getTask().forEach((task) => {
+        if (taskList.dataset.indexNumber === task.index) {
+          task.description = taskList.value;
+          localStorage.setItem('Task-list', JSON.stringify(getTask()));
+        }
+      });
+    });
+  });
+});
+
+window.addEventListener('load', () => {
+  updateTask();
+  dragDrop();
+});
+
+reload.addEventListener('click', () => {
+  location.reload();
```

Comment:
Using reload on a client-side JS app may slow down the response time of your project, so I suggest re-organising through DOM manipulation as even though it's also taxing on the browser, it still doesn't use as much as a full reload, and may improve the app's performance.